src/main/views/primary-health-list.njk

```diff
@@ -0,0 +1,104 @@
+          <P>{{ importantInformationP5 | replace("EMAIL", venueEmail) }}</P>
+          <P>{{ importantInformationP6 }}</P>
+        {% endif %}
+      </div>
+    </details>
+
+    <div class="parent-box overflow-table">
+      <table class="govuk-table">
+        <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header">{{ hearingDate }}</th>
+          <th scope="col" class="govuk-table__header">{{ caseName }}</th>
+          <th scope="col" class="govuk-table__header">{{ duration }} </th>
+          <th scope="col" class="govuk-table__header">{{ hearingType }}</th>
+          <th scope="col" class="govuk-table__header">{{ venue }}</th>
+        </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+        {% for courtCase in listData %}
+          {% set duration = '' %}
+          {% if courtCase['durationAsDays'] !== 0 %}
+            {% if courtCase['durationAsDays']  > 1 %}
+              {% set duration = courtCase['durationAsDays'] + ' ' + days %}
+            {% else %}
+              {% set duration = courtCase['durationAsDays'] + ' ' + day %}
+            {% endif %}
+          {% else %}
+            {% set durationAsHours = 0 %}
+            {% if courtCase['durationAsHours'] !== 0 %}
+              {% if courtCase['durationAsHours'] > 1 %}
+                {% set durationAsHours = courtCase['durationAsHours'] + ' ' + hours %}
+              {% elif courtCase['durationAsHours'] === 1 %}
+                {% set durationAsHours = courtCase['durationAsHours'] + ' ' + hour %}
+              {% endif %}
+            {% endif %}
+            {% set durationAsMinutes = 0 %}
+            {% if courtCase['durationAsMinutes'] !== 0 %}
+              {% if courtCase['durationAsMinutes'] > 1 %}
+                {% set durationAsMinutes = courtCase['durationAsMinutes'] + ' ' + mins %}
+              {% elif courtCase['durationAsMinutes'] === 1 %}
+                {% set durationAsMinutes = courtCase['durationAsMinutes'] + ' ' + min %}
+              {% endif %}
+            {% endif %}
+            {% if durationAsHours === 0 and durationAsMinutes === 0 %}
+              {% set duration = '' %}
+            {% elif durationAsHours !== 0 and durationAsMinutes !== 0 %}
+              {% set duration = durationAsHours + ' ' + durationAsMinutes %}
+            {% elif durationAsHours === 0 %}
+              {% set duration = durationAsMinutes %}
+            {% elif durationAsMinutes === 0 %}
+              {% set duration = durationAsHours %}
+            {% endif %}
+          {% endif %}
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell">{{ courtCase['hearingDate'] }}</td>
+            <td class="govuk-table__cell">{{ courtCase['caseName'] }}</td>
+            <td class="govuk-table__cell">{{ duration }}</td>
+            <td class="govuk-table__cell">{{ courtCase['hearingType'] }} [{{ courtCase['caseSequenceIndicator'] }}]</td>
```

Comment:
Sorry the case sequence indicator is normally optional so you don't want to show just the square brackets if it is missing You might want to add an if/else statements for this case. It is a bit ugly but will do the job :) 

{% if courtCase['caseSequenceIndicator'] | length %}
  <td class="govuk-table__cell">{{ courtCase['hearingType'] }} [{{ courtCase['caseSequenceIndicator'] }}]</td>
{% else %}
  <td class="govuk-table__cell">{{ courtCase['hearingType'] }}</td>
{% endif %}

Reply:
Sorted